macros/base/start_print.cfg

```diff
@@ -325,9 +347,11 @@ gcode:
 
     # Load intial tool if needed
     {% if ercf_enabled %}
-        ERCF_RESET_STATS
-        ERCF_CHANGE_TOOL_STANDALONE TOOL={INITIAL_TOOL}
-        _DO_ERROR_ERCF_PAUSE
+        {% if ercf_reset_stats_on_start_print %}
+            ERCF_RESET_STATS
+        {% endif %}
+        ERCF_CHANGE_TOOL STANDALONE=1 TOOL={INITIAL_TOOL}
+        _DO_ERROR_ERCF_PAUSE 
```

Comment:
Can this macro be renamed in: `_ERCF_ERROR_CHECK` ? It would help understand the problem and feature adressed by this custom macro